### PR TITLE
Added Text Expander v7.2

### PR DESCRIPTION
**Pull request by licon4812**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65116)

### CODE REVIEW

File: manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Smile.TextExpander
+PackageVersion: "7.2"
+PackageLocale: en-US
+Publisher: Smile
```
Comment (ItzLevvie) [inline manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.locale.en-US.yaml:7]:
```suggestion
Publisher: Smile, Inc.
```

File: manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Smile.TextExpander
+PackageVersion: "7.2"
```
Comment (ItzLevvie) [inline manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.installer.yaml:5]:
```suggestion
PackageVersion: "254.7.2.16"
```
This must match Control Panel.

You'll also need to update the version number in the package version folder to reflect this.

File: manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Smile.TextExpander
+PackageVersion: "7.2"
```
Comment (ItzLevvie) [inline manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.yaml:5]:
```suggestion
PackageVersion: "254.7.2.16"
```
This must match Control Panel.

You'll also need to update the version number in the package version folder to reflect this.

File: manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Smile.TextExpander
+PackageVersion: "7.2"
```
Comment (ItzLevvie) [inline manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.locale.en-US.yaml:5]:
```suggestion
PackageVersion: "254.7.2.16"
```
This must match Control Panel.

You'll also need to update the version number in the package version folder to reflect this.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**